src/views/FSL/renewal/renewalRevamped.vue

```diff
@@ -384,13 +389,12 @@
 
 				// localStorage.setItem('activeTab', tab.alias);
 				this.searchQuery.tab = tab.alias;
-				if(tab.alias !== "all"){
+				if(this.pageParams.page === 1){
```

Comment:
what's the purpose of this check?

Reply:
if you are in page 5 in 'all' tab and you go to contacted tab, it will get you back to page 1. But if it is a refresh and not tab switching, I assume you are not in page 1 and it will work. I will explain better in standup